### DISCUSSION TITLE
Printing dialog stalls due too high sheet count

### DISCUSSION THREAD

**Discussion by hroemer** [pradyunsg/furo#352] [Potential Issue] [answered]
Printing of a selected page does not work because the dialog stalls due to huge sheet count:

![print_dialog](https://user-images.githubusercontent.com/16097755/148642365-0bb7bfce-1c95-4678-9a87-4f3eb233dd85.png)

16.678 sheets for an empty default page created with `sphinx-quickstart`. No other extensions are loaded, I just switched theme to `furo`.

I also tried adding CSS media queries `@media print` from other themes but unfortunately with no luck.

Any ideas?

**Comment by pradyunsg** [comment]
What browser? Also, can you host this site somewhere (like GitHub Pages), along with the source code for generating it? I'm unable to reproduce this.

**Reply by hroemer** [reply]
The problem occurs under chromium v96. Printing dialog works well when using `sphinx-book-theme`, though.
I pushed a repro project to [hroemer/fury](https://github.com/hroemer/fury) and the build results are located inside the `docs` folder. (Sorry, I am not familiar with GH pages and currently could not get static content to load, yet.)

**Reply by hroemer** [reply]
OK, I figured out the `.nojekyll` workaround and now the pages are fully delivered.

* [fury/sphinx_book_theme/](https://hroemer.github.io/fury/sphinx_book_theme/index.html)
* [fury/furo/](https://hroemer.github.io/fury/furo/index.html)

**Reply by pradyunsg** [reply]
Ok, I can reproduce this on Chrome; but not Firefox or Safari. This is super weird.

**Reply by hroemer** [reply]
I have retested today and it does not occur anymore!?
Maybe it was fixed by a chromium update (`99.0.4844`) or was it part of furo `2022.3.4` and I missed it?

**Comment by pradyunsg** [comment, marked as answer]
Fixed in https://github.com/pradyunsg/furo/pull/319, by @jdknight! ❣️🎉
